TaskManager/TaskTableViewCell.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TaskTableViewCell.swift
+//  TaskManager
+//
+//  Created by user on 14.09.2022.
+//
+
+import UIKit
+
+class TaskTableViewCell: UITableViewCell {
+
+   
+    @IBOutlet weak var circleImageView: UIImageView!
+    @IBOutlet weak var firstTextView: UITextField!
+    @IBOutlet weak var secondTextView: UITextField!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        circleImageView.clipsToBounds = true
+        circleImageView.layer.cornerRadius = circleImageView.frame.size.width / 2
+    }
```

Comment:
in this cell you can create a function in which you will pass some params like text, isActive etc. This way you will be able to configure it based on if the task is active or not